src/js/components/stock-movement/outbound/StockMovementOutboundTable.jsx

```diff
@@ -207,6 +207,19 @@ const StockMovementOutboundTable = ({
       sortable: false,
       Cell: row => (<TableCell {...row} defaultValue="None" />),
     },
+    {
+      Header: <Translate id="react.stockMovement.request.approvers.label" defaultMessage="Approvers" />,
+      accessor: 'approvers',
```

Comment:
I think we shouldn't mix single quotation marks with double

Reply:
This is the eslint rule we have configured, we use single quotes for string values of an object and double quotes for attributes string values